README.md

```diff
@@ -40,72 +40,82 @@ Although experience with these technologies is not a requirement to join our tea
 able to work in this ecosystem. Your own experience will be taken into account when reviewing your assignment, so no
 need to panic if you are not familiar with them.
 
-You can complete the assingment using the following ecosystems:
+You can complete the assignment using the following ecosystems:
 
 - Node JS/TS
 - .NET (Core) 5+
 - Java / Kotlin
+- Python
```

Comment:
😞 😢 😱

Reply:
ODIN is written in Python, so I agree with Caio that we should allow the language in the assignment as well

Reply:
I actually completely missed that part when I first reviewed it.
The original idea we had with Lukas was that a we need people who can adapt to a new eco system. Since we're using Typescript internally, we wanted to force the assignment to be in TS. Hence having a skeleton, to reduce the overhead of having to setup everything if you're not familiar with it.

In this light I would either allow any language (why can't they do it in C++ if they want to?) or only allow TS (and maybe allow any language for more Junior positions, so they are more comfortable).

Reply:
If this is something we don't want, then I can remove it, but maybe lets discuss it or rewrite it a bit (where we state that we'd rather you do it in JS/TS).

To me, ideally the candidate should deliver using the assignment using the base to become familiar with the stack, this should also help us have a nice baseline to evaluate how things were done. 

Also, since "time" is precious during the hiring process, sometimes the candidate won't continue due to the fact that they'd rather do it in a stack they are most familiar with to be able to continue multiple processes, and by not allowing them to do it this way we risk losing a potential candidate that has too many processes going on (it happened already last week).
Sometimes their expertise doesn't transition immediately, but the engineering principles are the same

Now for the C/C++ part, do we have people that can and are willing to evaluate?

I can evaluate in JS/TS, Java, Kotlin, C# while others can also evaluate in Python (this was included because we are also looking for a new joiner for `spops`, which would need some snakezzz language knowledge

Reply:
Yeah that's the other side of the coin. Maybe at least stating that TS is preferred as this is what we use, and that we would take it into account if they have no or little prior knowledge of the eco-system, so they should not hesitate to give it a try if they feel like it.
I'm a bit rusty (uhuh) but I can probably still evaluate stuff in C++. Also in Python btw.

Reply:
Done and added a little disclaimer to the langgages/ecosystems/stacks